src/Components/CameraFeed/CameraFeed.tsx

```diff
@@ -28,6 +28,8 @@ interface Props {
   shortcutsDisabled?: boolean;
   onMove?: () => void;
   operate: ReturnType<typeof useOperateCamera>["operate"];
+  //Assest Configuration
+  isAssetConfigurationPage?: boolean;
```

Comment:
This prop seems very rigid, make it a bit generalized, something like showMonitorAsset or hideMonitorAsset.

Reply:
I have done the requested changes but now some tests are failing.